jetty-core/jetty-openid/src/main/config/modules/openid.mod

```diff
@@ -12,7 +12,7 @@ lib/jetty-util-ajax-${jetty.version}.jar
 lib/jetty-openid-${jetty.version}.jar
 
 [files]
-basehome:modules/openid/jetty-openid-baseloginservice.xml|etc/jetty-openid-baseloginservice.xml
+basehome:etc/jetty-openid-baseloginservice.xml|etc/jetty-openid-baseloginservice.xml
```

Comment:
ugh, don't copy a live XML from jetty-home to jetty-base.
The modules/openid/ directory exists for a reason.

Reply:
I thought it was useful, because if the xml was deleted from jetty-base/etc then it still works.
And the use of this XML is optional, but it would be annoying to have to manually copy it over when you need to use it.

Anyway I reverted this change so the file is back at `modules/openid/jetty-openid-baseloginservice.xml`.